readme.md

```diff
@@ -5,9 +5,11 @@ Seen these cool projects on github? Pain to download? This program is for you!
 
 Installation
 ------------
-If only there were some easy way to download this program from github...
+The easiest way to install this project on your computercraft computer is by running:
 
-Copy both files in `apis/` to `/apis/`, and put `programs/github` wherever you want it.
+    pastebin run p8PJVxC4 [<branch>|<tag>]
```

Comment:
As an example to (c) in my previous comment:

computercraft-github is modified and now has a file called `programs/foo`.  In order for `install.lua` to work properly we would need to add `programs/foo` to the `FILES` table.  But `programs/foo` uses some non backwards compatible features introduced in CC x.x.x designed for minecraft 10.2.  I'm playing a pack that is off minecraft 10.1... If I pull the trigger on `master/install.lua` and then try to execute github to download the version I want it fails because `programs/foo` is using api commands that don't exist in my version of CC.  So instead I can see that you add a tag to your project for a version of comutercraft-github that works with my version of CC.  I can now simply state `pastebin run :id version-that-works-with-1.7-github` and go on my marry way.  This feature is not only necessary to test this PR before we merge it into master (since install.lua doesn't even exist on master atm), it's wholly beneficial to provide backwards compatibility support.